packages/assets/scss/03-organisms/_inline-links.scss

```
@@ -0,0 +1,79 @@
+@use "00-base/configure" as *;
+
+.ma__inline-links {
+  @media print {
+    display: none;
+  }
+
+  &__container {
+    @include ma-container;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: stretch;
+  }
+
+  &__item {
+    font-size: $fonts-xsmall;
+    display: flex;
+    align-items: center;
+    min-height: 1.9rem;
+
+    &:after {
+      content: "";
+      margin: 0 10px;
+      width: 1px;
+      height: $fonts-xsmall;
+    }
+
+    &:last-of-type:after {
+      content: none;
+    }
+
+    a {
+      line-height: 1rem;
+      text-decoration: none;
+      box-shadow: inset 0 0 white, inset 0px -1px rgba($c-primary, 0.4);
+
+      &:hover {
+        box-shadow: inset 0 0 white, inset 0px -2px rgba($c-primary, 0.7);
+      }
+    }
+
+    span {
+      white-space: nowrap;
+    }
+  }
+}
+
+//theme
+
+.ma__inline-links {
+  &__item {
+
+    &:after {
+      background: $c-gray-light;
+    }
+
+    a {
+      font-weight: $fonts-normal;
+    }
+
+    span {
+      color: $c-font-detail;
+    }
+  }
+
+  &--alternative {
```

```suggestion
  &--alternative {
```
seem comment below about the naming

---

done